pkg/reconciliation/reconcile_racks.go

```diff
@@ -2243,6 +2246,52 @@ func (rc *ReconciliationContext) fixMissingPVC() (bool, error) {
 	return false, nil
 }
 
+func (rc *ReconciliationContext) SetFullQueryLogging() result.ReconcileResult {
+	datacenter := rc.GetDatacenter()
+	var DCConfig map[string]interface{}
```

Comment:
Wrong visibility for DCConfig, start with lowerCase.

Reply:
Would visibility not be confined to the function given that this is a variable locally instantiated there? Or would this cause it to leak out into the scope of `rc` because this function has it as a receiver?

Either way I think your suggestion is more idiomatic and will implement.